src/test/java/com/kodilla/ecommercee/domain/GroupTest.java

```diff
@@ -0,0 +1,89 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class GroupTest {
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Test
+    public void testSave() {
+        //Given
+        Group group = groupRepository.save(new Group(null, "Chemia"));
+        //When
+        long size = groupRepository.count();
+        //Then
+        assertEquals(1L, size);
+    }
+
+    @Test
+    public void testSaveWrongData() {
+        //Given
+        Exception exception = null;
+        //When
+        try {
+            Group group = groupRepository.save(new Group(null, null));
```

Comment:
Jeżeli dalej nie korzystasz w ogóle ze zmiennej `group` to można pominąć zapisywanie do niej wyniku.